templates/java/build-alt.gradle.mustache

```diff
@@ -28,8 +28,9 @@ repositories {
 }
 
 dependencies {
-  compile "com.google.protobuf:protobuf-java:3.0.0-beta-2"
+  compile "com.google.protobuf:protobuf-java:3.0.0-beta-1"
```

Comment:
Change to 3.0.0-beta-2 since we are using io.grpc:grpc-all:0.14.1 here.

Other things looks good to me.